examples/cstg/html/index.html

```diff
@@ -19,7 +19,7 @@
         $('#targeted_advertising_ready').text(__uid2.getAdvertisingToken() ? 'yes' : 'no');
         $('#advertising_token').text(String(__uid2.getAdvertisingToken()));
         $('#login_required').text(
-          __uid2.isLoginRequired() || __uid2.isLoginRequired() === undefined ? 'yes' : 'no',
+          __uid2.isLoginRequired() || __uid2.isLoginRequired() === undefined ? 'yes' : 'no'
```

Comment:
@lionell-pack-ttd I swear I did not change this line. This and line 1... I saw these mods, discarded changes, then made my line 78 change again and these two mods still showed up. For your consideration.

Reply:
The comma shouldn't be there - VS code is right to remove it :)